openmls/src/extensions/mod.rs

```diff
@@ -20,7 +20,7 @@
 //! - [`ExternalPubExtension`] (GroupInfo extension)
 
 use serde::{Deserialize, Serialize};
-use std::{convert::TryFrom, fmt::Debug};
+use std::{collections::BTreeSet, convert::TryFrom, fmt::Debug};
```

Comment:
While you're here you can drop the `TryFrom`.

Reply:
Ahh... Rust 2021 has `TryFrom` in the prelude? I didn't know :-)